web/src/components/planner/products/Product.vue

```diff
@@ -297,4 +338,7 @@
   .product {
     border-left: 5px solid #2196f3 !important
   }
+  span.v-chip.blue div.v-input:deep(input) {
```

Comment:
Would much prefer a custom class on this one :)

Reply:
Something like this?

Reply:
Was probably better than my way of doing it haha as it's more specific so yes, happy :)